### DISCUSSION TITLE
Sonoff Zigbee Dongle is called wrong in my raspberry

### DISCUSSION THREAD

**Discussion by Jochbart** [Koenkk/zigbee2mqtt#11404] [Devices]
Hi, 
if I am looking for the Sonoff Dongle, I do not get the name of it: 

pi@raspberrypi:~ $ ls /dev/serial/by-id
usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_16b743049c93eb11a4d4214f3d98b6d1-if00-port0 

Is there something wrong with my dongle, or why is there not the actual name of it? The dongle also does not appear in my frontend as my first device. Thanks for helping.

**Comment by francisp2** [comment]
You simply have a stick from the first 2 batches. Mine are called something like that too.
